desktop/components/auction_block/index.jsx

```diff
@@ -0,0 +1,25 @@
+import AuctionBlock from './auction_block.jsx'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { isEmpty, isString } from 'underscore'
+
+export default function mount (data, selector = '#react-mount-current-auctions') {
+  const {
+    relatedAuction,
+    sale
+  } = data
+
+  const shouldMount = !isEmpty(sale.id) && isString(selector)
+
```

Comment:
This is something I've been meaning to do in `current_auctions` but haven't gotten around to it yet (do you think you could update for me?) -- add in a runtime invariant (which just passes right through if truthy, but warns if false) that warns the user if something is up with their props:

```javascript
import invariant from 'invariant' 
... 
const shouldMount = !isEmpty(sale.id) && isString(selector)

invariant(shouldMount, 
  `Error mounting <AuctionBlock />: sale.id (${sale.id}) or `selector` (${selector}) is invalid.`
)
````
Or something like the above; could even move it into an else at the bottom -- something to alert devs that something is wrong with the operation / `shouldMount` is false.)